stack.yaml

```diff
@@ -54,6 +54,9 @@ extra-deps:
   - typerep-map-0.6.0.0
   - monad-validate-1.2.0.1
 
+# TODO remove when co-log is updated
+allow-newer: true
+
```

Comment:
This was removed by #3612 but is apparently required for `co-log`  (for the `typerep-map` dependency, to be precise). The new `co-log` release should come soon, then we can delete it.

Q is, why did the CI build not pick this up in the other PR? Probably due to caching.